### ISSUE TITLE
Resisted wrist extension provokes pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10808]
**Preferred term label:**
Resisted wrist extension provokes pain

**Synonyms**

**Definition (free text, please give PubMed ID)**
Active wrist extension against resistance increases pain in the arm

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0030836

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Extensor digitorum communis tenosynovitis
Lateral epicondylitis
Intersection syndrome of hand tendons

**Your nano-attribution (ORCID)**
